applications/workspaces/api/openapi.yaml

```diff
@@ -18,13 +18,43 @@ tags:
 - description: Client
   name: Client
 paths:
+  /live:
+    get:
+      summary: Test if application is healthy
+      security: []
+      operationId: live
+      tags:
+        - k8s
+      responses:
+        "500":
+          description: Application is not healthy
+        "200":
+          description: Healthy
+          content:
+            application/json:
+              schema:
+                type: string
+  /ready:
+    get:
+      summary: Test if application is ready to take requests
+      security: []
+      operationId: ready
+      tags:
+        - k8s
+      responses:
+        "500":
+          description: Application is not ready yet
+        "200":
+          description: Ready
+          content:
+            application/json:
+              schema:
+                type: string
   /workspace:
     get:
       tags:
         - rest
       summary: Used to list all available workspaces.
-      security:
```

Comment:
why remove the security

Reply:
because we can get public workspaces if not logged. So the api is not requiring authentication to be accessed.

Reply:
ah yes correct, nice catch